.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  "configurations": [
```

Comment:
Is it intended to push this?

Reply:
Yes, just to launch the NestJS project in debug mode, with breakpoints and all